galasa-parent/dev.galasa.framework.api.resources/src/main/java/dev/galasa/framework/api/resources/ResourcesServlet.java

```diff
@@ -50,8 +51,12 @@ public void init() throws ServletException {
 
 		super.init();
 
-		addRoute(new ResourcesRoute(getResponseBuilder(),framework));
-
+		try {
+            addRoute(new ResourcesRoute(getResponseBuilder(), framework));
+        } catch (ConfigurationPropertyStoreException e) {
+            throw new ServletException("Failed to initialise Resources Servlet", e);
```

Comment:
Does this neeed logging also ? or is throwing the exception enough that it appears in the log ?

Reply:
Added log message in https://github.com/galasa-dev/framework/pull/638/commits/e0ebeed9d20a210f1213aa84a9603a7cd396b364